doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+
+### Setup
+
+**Minimum requirements:** Materialize v0.8.0+, Postgres 10+
+
+Before creating a source in Materialize, you need to ensure that the upstream database is configured to support [logical replication](https://www.postgresql.org/docs/10/logical-replication.html). As a _superuser_:
+
+1. Check the [`wal_level` configuration](https://www.postgresql.org/docs/current/wal-configuration.html) setting:
+
+    ```sql
+    SHOW wal_level;
+    ```
+
+    The default value is `replica`. For CDC, you'll need to set it to `logical` in the database configuration file (`postgresql.conf`). Keep in mind that changing the `wal_level` requires a restart of the Postgres instance and can affect database performance.
+
+    **Note:** Additional steps may be required if you're using Postgres on [Amazon RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.FeatureSupport.LogicalReplication) or [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance).
+
+1. Grant the required privileges to the replication user:
+
+    ```sql
+    ALTER ROLE "user" WITH REPLICATION;
+    ```
+
+    **Note:** This user also needs `SELECT` privileges on the tables you want to replicate, for the initial table sync.
+
+1. Set the replica identity value for the tables you want to replicate:
+
+    ```sql
+    ALTER TABLE repl_table REPLICA IDENTITY FULL;
+    ```
+
+    This setting determines the amount of information that is written to the WAL in case of `UPDATE` and `DELETE` operations. By setting it to `FULL`, change events will include the previous values of all the table’s columns; as a heads-up, you should expect a performance hit in the database from increased CPU usage.
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    _For specific tables:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
+    ```
+
+    _For all tables in Postgres:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR ALL TABLES;
+    ```
+
+    The `mz_source` publication contains the set of change events generated from `table1` and `table2`. This publication will later be used to ingest the replication stream into Materialize.
+
+### Create a source
+
+Postgres sources are materialized by default and ingest the raw replication stream data for all tables included in the publication, to avoid creating multiple replication slots and minimize the required bandwidth. To create a source:
+
+```sql
+CREATE MATERIALIZED SOURCE mz_source
+FROM POSTGRES
+  CONNECTION 'host=postgres port=5432 user=host dbname=postgres sslmode=require'
```

Comment:
```suggestion
  CONNECTION 'host=example.com port=5432 user=host dbname=postgres sslmode=require'
```

`postgres` is a docker-ism that people following along might not know. Let's use `example.com` to show it's a domain name